spacelift/data_aws_integrations.go

```diff
@@ -0,0 +1,109 @@
+package spacelift
+
+import (
+	"context"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/spacelift-io/terraform-provider-spacelift/spacelift/internal"
+	"github.com/spacelift-io/terraform-provider-spacelift/spacelift/internal/structs"
+	"github.com/spacelift-io/terraform-provider-spacelift/spacelift/internal/validations"
+)
+
+func dataAWSIntegrations() *schema.Resource {
+	return &schema.Resource{
+		Description: "`spacelift_aws_integration` represents a list of all the AWS integrations in the Spacelift account",
```

Comment:
in the Spacelift account -> in the Spacelift account visible to the API user. The reason is with spaces you might see just a subset.